app/models/runtime/process_model.rb

```diff
@@ -184,7 +184,7 @@ def package_hash
       return nil if package.nil?
 
       if package.bits?
-        package.package_hash
+        package.checksum_info[:value]
       elsif package.docker?
         package.image
```

Comment:
The fact that a method called `package_hash` can return a docker image address makes me not worried that it can now return a sha1 or a sha256 checksum.